### DISCUSSION TITLE
Consider using libfyaml?

### DISCUSSION THREAD

**Discussion by emiltin** [ruby/psych#637] [Ideas]
It seems libyaml is ney yet fully compatible with the YAML 1.2 spec.
Did you consider using libfyaml? https://github.com/pantoniou/libfyaml I have no experience with this lib, but it claims to be fully 1.2 compatible, using the official test suite from  https://github.com/yaml/yaml-test-suite.

**Comment by hsbt** [comment]
psych 5.5.0 ships an experimental libfyaml backend, enabled with `gem install psych -- --enable-libfyaml`. The default stays libyaml, and #584 tracks YAML 1.2 by default.
